app/src/main/java/org/systers/mentorship/view/activities/SettingsActivity.kt

```diff
@@ -42,6 +43,20 @@ class SettingsActivity : BaseActivity() {
         tvAbout.setOnClickListener {
             startActivity(Intent(baseContext,AboutActivity::class.java))
         }
+        tvDeleteAccount.setOnClickListener {
+            val builder = AlertDialog.Builder(this)
+            builder.setTitle(R.string.delete_account)
+            builder.setMessage(R.string.delete_account_message)
+            builder.setPositiveButton(R.string.delete) { _, _ ->
+                //@todo any function here
+                Toast.makeText(applicationContext,"Account deleted", Toast.LENGTH_SHORT).show()
```

Comment:
Add this string "Account Deleted" in strings.xml file

Reply:
> Thanks for your contribution @amosgamaliel 👍 Great work so far 🎉
> You just need to do some changes -
> 
> * Change the commit message to follow [Commit Message style guidelines](https://github.com/systers/mentorship-android/wiki/Commit-Message-Style-Guide) and change it to something like **feat: Delete account confirmation dialog**
> * Redirect this PR to **gci-dev** branch instead of **develop**

thank you sir, i will do your suggestion
gbu